Rakefile

```diff
@@ -16,9 +16,17 @@ end
 
 require_relative 'lib/audit/moab_to_catalog'
 desc 'populate the catalog with the contents of the online storage roots'
-task seed_catalog: :environment do
+task :seed_catalog, [:profile] => [:environment] do |_t, args|
+  unless args[:profile] == 'profile' || args[:profile].nil?
+    p "Usage: rake seed_catalog || rake seed_catalog[profile]"
+    exit
+  end
   m2c = MoabToCatalog.new
   puts "#{Time.now.utc.iso8601} Seeding the database from all storage roots..."
```

Comment:
might be nice to indicate in the `puts` msg that the run is being done with profiling, but not a big deal, and happy to ticket that as follow up if you do agree it'd be a good idea.

Reply:
I think indicating profiling or not in the puts message is a great idea.

Reply:
Thanks @jmartin-sul I'm agreed they're good suggestions, and I took a stab at both.